lib/ovirt/logging.rb

```diff
@@ -0,0 +1,5 @@
+module Ovirt
+  module Logging
+    delegate :logger, :to => :Ovirt
```

Comment:
How does it work?  Logging delegates logger to Ovirt, but the logger method on Ovirt is an instance method...should it be a class method?